README.md

```diff
@@ -1,151 +1,99 @@
-
-Default administrator and YunoHost users must login using LDAP:
-* click on the "with LDAP" button
-* use your YunoHost credentials
-
-When disabling Local Login and other authentication services, clicking the `Login` or `Sign Up` button will directly bring up the LDAP Login popup.
-
-![Disable Local](https://raw.githubusercontent.com/jonmbake/screenshots/master/discourse-ldap-auth/disable_local.png)
-
-![LDAP Login Popup](https://raw.githubusercontent.com/jonmbake/screenshots/master/discourse-ldap-auth/ldap_popup.png)
-
-#### Supported architectures
-
-* x86-64 - [![Build Status](https://ci-apps.yunohost.org/ci/logs/discourse.svg)](https://ci-apps.yunohost.org/ci/apps/discourse/)
-* ARMv8-A - [![Build Status](https://ci-apps-arm.yunohost.org/ci/logs/discourse.svg)](https://ci-apps-arm.yunohost.org/ci/apps/discourse/)
-
-## Limitations
-
-None at the moment.
-
-## Additional information
-### Known non-impacting log messages
-```
-fatal: Not a git repository (or any of the parent directories): .git
-
-bash: BASH_XTRACEFD: 7: invalid value for trace file descriptor
-```
-## How-tos
-### Install plugins
-```
-cd /var/www/discourse
-sudo -i -u discourse RAILS_ENV=production bin/rake --trace plugin:install repo=https://github.com/discourse/discourse-solved (for example)
-sudo -i -u discourse RAILS_ENV=production bin/rake --trace assets:precompile
-systemctl restart discourse
-```
-
-## Links
-
- * Report a bug: https://github.com/YunoHost-Apps/discourse_ynh/issues
- * App website: https://discourse.org/
- * Upstream app repository: https://github.com/discourse/discourse
- * YunoHost website: https://yunohost.org/
-
----
-
-## Developer info
-
-Please send your pull request to the [testing branch](https://github.com/YunoHost-Apps/discourse_ynh/tree/testing).
-
-To try the testing branch, please proceed like that.
-```
-sudo yunohost app install https://github.com/YunoHost-Apps/discourse_ynh/tree/testing --debug
-or
-sudo yunohost app upgrade discourse -u https://github.com/YunoHost-Apps/discourse_ynh/tree/testing --debug
-```
+<!--
+N.B.: This README was automatically generated by https://github.com/YunoHost/apps/tree/master/tools/README-generator
+It shall NOT be edited by hand.
+-->
+
+# Discourse for YunoHost
+
+[![Integration level](https://dash.yunohost.org/integration/discourse.svg)](https://dash.yunohost.org/appci/app/discourse) ![](https://ci-apps.yunohost.org/ci/badges/discourse.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/discourse.maintain.svg)  
+[![Install Discourse with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app=discourse)
+
+*[Lire ce readme en français.](./README_fr.md)*
+
+> *This package allows you to install Discourse quickly and simply on a YunoHost server.
+If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/install) to learn how to install it.*
+
+## Overview
+
+Discussion platform
+
+**Shipped version:** 2.8.0~ynh1
```

Comment:
This is actually a beta version (2.8.0.beta1), we'd better be precise in the YunoHost package version, because there will be updates.

Reply:
You're right ... we could just increment to 2.8.0~ynh2 for final release (i don't think the regex accepts alphabetic characters for the app version.)
(I set the version to 2.8.0 beta1 because I notice it was the version used by yunohost.forum.org)